packages/frontend/config-overrides.js

```diff
@@ -1,10 +0,0 @@
-const { alias } = require("react-app-rewire-alias");
-
-module.exports = function override(config) {
-  alias({
-    common: "../common",
-    "@common": "common",
-  })(config);
-
-  return config;
-};
```

Comment:
do we not need this anymore??

Reply:
if we don't know whether we need it or not/are confused, no point in doing research let's add it back and not touch the old frontend lmaooo

Reply:
we don't need it because we're now using Create react app v5